### ISSUE TITLE
Don't understand?

### ISSUE THREAD

**Issue by Berzetoberze** [Hokan-Ashir/SFGameDataEditor#8]
Hi Hokan Ashir, The Great Necromancer of Eo.

I need help about your amazing editor!

How can we change .mod to .cff or .sfmod to .cff because I have soo many copies of them but I can't use it? Have I misunderstand some steps of your software?

Some bugs detected or work in progress:
- French translation is not very good some spell are named weirdly... Like Fireburst are become Firecrash...  
- Some Spell like Lifetap Aura have ??? is their parameters
- Some units cannot learn some spell like Havoc (Dark Elves) units cannot learn Battle Cry or Resist

Thanks again for your future answer!

**Comment by Berzetoberze** [comment]
Oh sorry I forgot my config!

I play on SpellForce Platinium Edition in French, patch 1.52, Windows 7, 32 bits.

**Comment by Hokan-Ashir** [comment]
Hi Berzetoberze (did I wore it right?)

It seems, that I didn’t write easy-to-go guide for it =)
Ok, that’s easy to get to.
You selected .cff file, did some changes, created mod file.
Mod file is essentially .cff + your changes. Plus, custom file extension, so that it accidentally don’t overwrite original .cff file
So to use .mod-file, just rename file and strip “.mod” at the end of file
You may not have ability to change file extensions natively, if you’re running Windows, just google how to do it, it’s several steps =)

About work in progress and bugs:
 - French translation is almost google-translate + a bit vocabulary and Russian friend that knows French but doesn’t know the game; so no surprise it’s awful, beside that, I changed almost all similar words with some replacements, that’s why “fire burst” is not any closer to “fueXXX”; if you would like to change French translation, please, feel like home - at https://github.com/Hokan-Ashir/SFGameDataEditor/tree/master/Editor/src/main/resources <https://github.com/Hokan-Ashir/SFGameDataEditor/tree/master/Editor/src/main/resources> you can see tuples of four files, that corresponds to localization;
i.e.
Armor_gui.properties (for armor GUI; English language)
Armor_gui_de.properties (German language)
Armor_gui_fr.properties (French language)
Armor_gui_ru.properties (Russian language)
So if you would like to change French translation - take English one, change text at the right side of equation symbol (“=“) and, if you want - create new issue via “New issue” button at https://github.com/Hokan-Ashir/SFGameDataEditor/tree/master/Editor/src/main/resources <https://github.com/Hokan-Ashir/SFGameDataEditor/tree/master/Editor/src/main/resources> if you have Github account, or send me new file, I’ll just commit it to repository 


- About “???” marks - in “How you can help” section on GitHub readme (https://github.com/Hokan-Ashir/SFGameDataEditor <https://github.com/Hokan-Ashir/SFGameDataEditor>) you can see that these parameters are unknown to me; some of them maybe left as deprecated, as game development progressed, some actually mean something; but I really dunno what exactly they are for. If you aware of any of them - please, as in previous case - either create new tickets in GitHub, or just write via email about what each parameter you know of describes

- Oh, man. That’s really pain in the ass =) In fact no one actually knows what will work during changes. I mean literally. For example - if you would like to change merchant’s items list, the actually GameData.cff file changes. But you will see no changes in the game. Why? No one knows. Some spell parameters really doesn’t affect spells, some item parameters doesn’t affect items (weapon material for example). Some creatures are not summonable via "SummonXXXX" spells
So you can create a list of what actually doesn’t work, describing what you did, what you’ve changed, and what you expected, I’ll try to systemize it somehow - maybe will make a file and place it into repository, maybe create new issue with “info” tag, but such things will be nice to remember and write down somewhere


Thanks, really thanks for your letter
Trulls warm my heart when somebody get to editor and begins to use it =)

**Comment by Berzetoberze** [comment]
Hi again! Really quick answer! And yes good writing ;) it's just Berze to berze easy right? :p

_So to use .mod-file, just rename file and strip “.mod” at the end of file You may not have ability to change file extensions natively, if you’re running Windows, just google how to do it, it’s several steps =)_
I really don't understand this part... When I will click on "create a .sf-mod file" the software shut down and I don't have .sf-mod file but if I understand well, the software make can the change on the .cff directly right?

- For the French translation, leave it to me, I'm glad to help! But I'm honest with you, I can't make it all very quickly, I'm slow like a turtle! Say just how translation you need in priorities and I will make it!
For the moment, I have translate all the "armor_gui_fr.properties"
- Hmm I see, I will make test if I can to say what is it...

- Hmm... I have try to level up Kyra (when she is a enemy not when is it in rune) at lv.20 and become immune to ice. But nothing happened... Or I really don't know how software works!

Oh soo sweet you are! ^^
You're editor is amazing! But I can't enjoy the fun for the moment. :p

**Comment by Hokan-Ashir** [comment]
Wow, just figured you’ve wrote actual issue request on GitHub =)
Ah, you’ve joined just 4 hours ago =) okay

Nice catch, you’ve just accidentally figured new issue - creation of sf-mod files isn’t working on new alpha version. That’s not a big deal, and you can work without them at all

Ok, if you didn’t read README.md on the main page of repository, I really advice you to do so - you’ll figure some restrictions that editor currently has and be aware of its possibilities
So, what actual schema of editor’s work:
You select Gamedata.cff file, edit it, then you have two options
1. Create “.mod” file. It’s Gamedata.cff + changes _in Gamedata.cff format_. So basically what you have to do to force it to work is just delete “.mod” extension at the end of file. That’s it. Gamedata.cff.mod rename to Gamedata.cff, place in WHERE_YOUR_SPELLFORCE_INSTALLED/data folder, replace file that already there and launch the game
2. Create “sf-mod” file. That’s a different story. This file contains only compressed changes. So it’s easy to share your changes sending a little file rather than whole Gamedata.cff, that you’ve changed. I designed this format keeping in mind players that would like to share their changes, creating paths to game and mods, but eventually, Editor has not so widespread so that anyone would like to use it =)


Nice! Really awesome! You can create new ticket, to merge changes that you’ve made, like you’ve already done with this conversation

And such “it should have worked! I’ve changed” behavior is quite natural, (not that you’ve mentioned, about Kyra) I really doubt that developers though that someone would like to summon “Bird” with SummonSkeleton - level 10 spell =)
Btw, try out, you’ll see what a “bird” it is =)

**Comment by Berzetoberze** [comment]
This is the first step when I got a software, I read immediately the "Read Me" and to be honest I have done soo many changes and nothing happened but yesterday night... Miracle! It's works! It's a such of non-sense! Whatever...

Now I can enjoy the fun and make my own SpellForce and having fun with them! ;)

And for the changes, I'm glad to help you, your software is very nice and for thanking you, I want to help with the French translations!
What kind of French translation you need first?

For the summon skeleton spell, I will try out after

**Comment by Hokan-Ashir** [comment]
This is awesome, glad to hear you've solved this little issue =))

Just send them in any order you like, it really doesn't matter what you
translate first, I'd like to hear any new stuff from you)

27 мая 2017 г. 3:27 ПП пользователь "Berzetoberze" <notifications@github.com>
написал:

**Comment by Berzetoberze** [comment]
Yes! It's great! I think you need to add in the "Read Me" to change .mod to .cff

For the translations, ok no problem, I will make it slowly but surely!

For the Skeleton spell, it's funny ^^ It's too bad he can't fight or is to be invincible... :/
Did you test to summon "Named Humans" "******"? Take a look!

Other problem spotted and suggestions:
- Some spell in your editor is in a wrong place like "Reinforcement" is White Magic: Boons not Black Magic: Curses
- Some pictures are in the wrong place. For example in the Trolls units, the "Smasher Upgrade" and "Bouncer Upgrade" are reverse.
- The software cannot save the file source and we need to search again everytime, it's a bit annoying... :/
- The window of the software are too big for a 1366x768 screen. Can you update this is possible? Because the first time I launch the editor, I don't find the three options for create and save the changes.
- Can we change the amount of units upgrading? I want to change the wooden shield of the Elves Guard in iron shield but I cannot change the type of resources needed. For the moment is 300 Wood and I want to change into 300 Iron.
- It's possible to change some buildings? Because Trolls and Elves need iron for some units but they haven't mine for extract iron! It's a kind of big problem right here!

Future suggestions and problem spotted will coming soon, be patient! ;)

**Comment by Hokan-Ashir** [comment]
Ah, sorry to disappoint you, but I can't test anything =) SpellForce isn't a good friend of Linux, or maybe I has some weirdo video card - have glitches, wrong textures, floating shadows, avatar itself seems like a bunch of planes sewed in one point on the ground

- This maybe confusing at start but that's really how actual data is stored inside Gamedata.cff. Aren't you surprised that there is no "Summon spider" spell but there is "Summon skeleton - level 101" spell, that summons spiders, which is used by Creatures/Goblins/Spiderwhisperer? =) That's developer's stuff and Editor's task is just forward that data in fancy GUI way. Another example is some spell's scrolls that are not sorted correctly in game, check out "Mana sacrifice - level 8 and 9" for example, it will be sorted between black magic spells, AFAIR. But you may change that as you wish =)

- That's may be possible, trolls all have the same faces =) Cause, as I mentioned above I can't launch the game, could you please write what pairs of creatures pictures must be swapped?
You may ask me, how I got creatures images without the actual game. That's work of one buddy in Russia, helped a lot, but wished to be anonymous =)

- You mean Editor can't refill initial paths to .cff and .sf-mod files on "File selection" window? Hmmm, that can be solved with a little blood =)

- I can play with it, forcing it to inherit system's width and height.

- That's a good question. There is really some info about upgrades that stored inside .cff, but I currently have no awareness of what exactly bytes corresponds to what. You can try to take a look at so called "buildings_2" bytes format here - http://spellforcefanforum.hostoi.com/viewtopic.php?f=29&t=316

- You can swap some "unnecessary" elves building with other building that can mine, but I'm not sure that's possible. And I've also tested - unfortunately it's impossible to add new records into .cff file - game either won't start or it will crush during map/saves loading

btw, thanks a ton for french translation and pull requests, I've commented couple of them, which doubted me (https://github.com/Hokan-Ashir/SFGameDataEditor/pull/14 and https://github.com/Hokan-Ashir/SFGameDataEditor/pull/10)

**Comment by Berzetoberze** [comment]
Sorry for the late answer, Github doesn't notice me... I don't know why?

You work on Linux? Oh it's a bit difficult isn't it?

- For the spell, yes, I see after posting my report. When I will translate, I will by careful about that and I make the change in the editor

- It's the only one, I think it was more but no just this one must be swapped.

- Yes that right, you must find everytime the file source of the .cff when you start the software.

- You cannot extend the window of the software? If you can't, you must notice to the "Read Me"

- Bad Gateway 502... ^^' an other link available?

- Oh... We have to deal with it, unfortunately... 

No problem, I will make the other surely but slowly when I said before.

Other problem spotted:

- Add to your "Read Me". You cannot change all stats of Belial because you cannot finish the game when you kill it and a "Fatal Error" will appear.

- Gonthar Thorme, had a wrong picture. The real picture of Gonthar is a Recruit not a Enchanter.

**Comment by Hokan-Ashir** [comment]
> You work on Linux? Oh it's a bit difficult isn't it?

Nope, easier than on Windows, but that's just my own preferences. And as side effect lack of real testing. Just like Linux systems, really easier for any development

> You cannot extend the window of the software? If you can't, you must notice to the "Read Me"

You misunderstood. It really possible - just extend whole application window to max width and height of user's screen

> Bad Gateway 502... ^^' an other link available?

That's weird. Try this one - http://spellforcefanforum.hostoi.com/viewforum.php?f=29&sid=f95a2e68d8995903211de86631a78d02 and then search for "25. building stats 2" thread

Okay, that's nice; I'll wrap your notices into GutHub issues

Keep finding new bugs, I'm going to massively fix them all at once, and make a 9-alpha-snapshot version
Maybe also add new features, like, as you mentioned, changing update-unit costs and additional images

Late answers alright - I don't have much time either =)

**Comment by Berzetoberze** [comment]
Ok I see, I don't know how work Linux soo I can't say anything on that! =p

Oh ok I will try after, I'm not in my home actually. Sometimes I don't understood well, my bad.

I know this site! It's very nice but I don't understand where applies these values and which software change it?

You can count on me! Other bugs spotted:

- A lot of pictures is missing for the Screamers and Luciens

- Briarwolves recruit had a wrong picture, is not a mentalist.

- Elves units cannot hold correctly a shield, it's very very annoying. They hold shields like a weapon... Very wierd when you see that. Can we fix it?

- Section protected guardians has a mistake you have wrote Protected guradians or something like that... But don't worry, I fix that!

- I have spotted others but I don't remember for the moment and when I said, I'm not in my home with my little paper where I wrote the bugs. Be patient my friend ;)

**Comment by Hokan-Ashir** [comment]
> It's very nice but I don't understand where applies these values and which software change it?

That's the hard part. It's basically just numbers, what actually they do no one have any idea, besides what already has been figured. One way to somehow solve it is just add more code into editor and left unknown values in a digit way. I mean, let's assume we can get some data about "Light's Chapel" (the building that allows humans to summon Clerics), I can map this data to this specific building and left something like "2343343" up to you. And you yourself will figure what is "2344434" means. How about that? =)

> Elves units cannot hold correctly a shield, it's very very annoying. They hold shields like a weapon... Very wierd when you see that. Can we fix it?

From Editor's perspective - no. That's either animation engine or model settings and assuming that in 2003 animation often were not separated, I guess the reason lies in *.obj files, that describes in-game models
Although, I can't remember, does OBJ-format store animations

> Section protected guardians has a mistake you have wrote Protected guradians or something like that... But don't worry, I fix that!

Not even me =) Just grabbed from those "hostoi"-site =), but thatnks anyway

Take your time =) You have all of it
I've merged other found bugs into issues and posted questions, as you've already noticed

**Comment by Berzetoberze** [comment]
I really don't understand this... ^^" need a explanation for dummies... =p

It's very bad... They can only hold correctly their respective shield but they are useless because these shield have 0 def and 0 stats...

Other bug spotted:
- We cannot edit orb item, it's a kind of bug? And we cannot edit the Null Item 3176 and 3177.

**Comment by Berzetoberze** [comment]
Another bug:
- We cannot change stats of weapon. For example, Everflame give 3 in Agility and we cannot change this, because the option is not proposed.

**Comment by Hokan-Ashir** [comment]
Double checked the link, that I've sent you - those info is about buildings geometry and didn't suit for our needs. Hmm, I'll dig into this forum once more and into my own computer, maybe I'll find units' upgrades Gamefata bytes range

> It's very bad... They can only hold correctly their respective shield but they are useless because these shield have 0 def and 0 stats...

But you can change their shield on any other shield and give them significant armour boost. Yeah, they won't look correctly and fancy, yet still that's an improvement

> We cannot edit orb item, it's a kind of bug? And we cannot edit the Null Item 3176 and 3177.

It is. =( I'll make a ticket for that. Thanks for spotting this out ^^

> We cannot change stats of weapon. For example, Everflame give 3 in Agility and we cannot change this, because the option is not proposed.

WoW. I almost forgot that some weapon actually has additional stats. Missed that during development. Also thanks again, "Eagle eye" =)

**Comment by Hokan-Ashir** [comment]
If you would like I can release alpha-9-snapshot with some images changes, fixed window size, added ability to change orb and weapon stats, for you to test

**Comment by Berzetoberze** [comment]
Yes I put shield to Protector elves but it's not look fancy... It's not bad

No problem my friend, I use a lot your software, so I spot quickly the bug and problems.

Yes why not? Release it ;)

Other bugs spotted:
-This is a big one...
All Titan Elves of the game (allies and enemies) has a wrong chest armour. They had "ItemChestFakeNPCUndeadGobelin" and I see a Hero Rune naked... It's very weird... Or they have "Ancient Dwarven Axe" we need to fix this before the release 9

I have found some ??? on spell parameters 

-  Extinct: is the range of the spell. Example, put it to 1000 and you can clean all the map in one hit... And you become God.

- Pain Tower: is the damage

- Extinct Tower: The first is the damage and the second the range

- Pain Area: the range

- Tower Stone: the damage

- Arrow Tour: the damage

- Black Protection: First the resistance, second the duration

- Stone Rain, Blizzard, Fire Rain: First and second, max to min range

**Comment by Hokan-Ashir** [comment]
> All Titan Elves of the game (allies and enemies) has a wrong chest armour. They had "ItemChestFakeNPCUndeadGobelin" and I see a Hero Rune naked... It's very weird... Or they have "Ancient Dwarven Axe"

"Ancient Dwarven Axe" is the first item in the list and in such a way signifies that creature object has no chest-object. I'll try to fix it later. Sorry, didn't catch up first part of your message about "ItemChestFakeNPCUndeadGobelin"

> Stone Rain, Blizzard, Fire Rain: First and second, max to min range

Is it "range" or "square range"? I.e. first parameter is "x"-axis range and second one is "y"-axis range
At "hostoi" site I came across such vision, yet have no possibility to test it
I'll add this description in such manner, yet I'd like you to double-check it

Added everything else in alpha-9-snapshot, take a look at https://github.com/Hokan-Ashir/SFGameDataEditor/tree/master/Editor/bin =)

**Comment by Berzetoberze** [comment]
Ok thanks for the Titan Elves ;)

Square range, sorry, forgot to tell you.

Now for your version 9... It's got some very annoying bugs:
- We cannot edit weapon at all, you select one weapon and nothing happens...

- I need to remake some of my French translation because sometimes the sentences is too long and we cannot see it in the editor.

- All words with an accent is translated by symbols... I have two solutions about that, choose one: 
First: I change all words with accent and I change the word with no accent. Example: é becomes e
Second: I try to find how to make disappear the symbols and put the real word with accent

Another things I have to tell you. This Friday, I'm in holidays, so I cannot see the changes, translate, misc... until September. Just be patient and after I will catch up my retard ;)

**Comment by Hokan-Ashir** [comment]
> Square range, sorry, forgot to tell you.

That's alright =)

> We cannot edit weapon at all, you select one weapon and nothing happens...

Yeah, you right, that's my poor build, cause in IDE everything works fine. The reason was in revamping WeaponsParametersView class/form

> All words with an accent is translated by symbols... I have two solutions about that, choose one:
First: I change all words with accent and I change the word with no accent. Example: é becomes e
Second: I try to find how to make disappear the symbols and put the real word with accent

My bad as well - didn't covert your merges into appropriate encoding

As for too narrow labels in any GUI element, my thoughts is to polish Editor look a like at very last step; can be fixed, but not in the highest priority

I've fixed it and you can check'em as new alpha-9 version

Okay =) Wish you a good holidays then, came back
Hope I'll finish adding images till that time

**Comment by Berzetoberze** [comment]
I have a bad news my dear... I'm very too busy this week before my departure. I can't help you for the moment. When I will in holidays, I only help you about the editor itself and I cannot test in the game, the improvement and bugs.
And thanks for wish me a good holidays! This is will be, because I will be with my lady again! ;)

**Comment by Hokan-Ashir** [comment]
That's alright, you can check at least some UI stuff and maybe misses during its configuration, that's more than sufficient =)

YAW, btw where are you going? If this isn't private information
// yeah, answering here, at open-source sharing platform, ahahah =)

**Comment by Berzetoberze** [comment]
Hey dear friend!

Sorry for the late answer! I'm too busy for the moment for impress my lady, go at beach, eat together, walk in love... And being a real duck... Hehehe =P

What am I going? For the moment, I'm at Nice in the South East of the France, near Italy. The weather is very hot but I love to hear cicada! It's very relaxing.

Apologies for not helping you for the moment. But I'm not in the mood, I just want to enjoy the holidays at 100%!

In September, when I coming back, I will help you again, don't worry. For the moment, just be patient, good luck and I give you my support in my mind ;)

**Comment by Hokan-Ashir** [comment]
Sure, man =) 
Won't bother you for some time 
And I'm pretty sure I'll need your help later on =) 

Happy lovely hot nights and days =)

**Comment by Berzetoberze** [comment]
Hey my dear friend! I'm back!

But I'm not ready yet for help you again. Please standby ;)

**Comment by Berzetoberze** [comment]
I have just downloaded the new release 9 of the editor but I have spotted a lot of bugs!

- Added ability to change weapon (like Everflame, Earthblade) stats
We cannot change... A lot of weapon was disappear of the editor, where are they? Example: I click on orb section, no screen appear or no item are found.

 - Added races images
Some are false, like Sha and others one but don't remember for now. I will check later.

 - Added switching localization in real time (a bit hanging of UI is acceptable)
Doesn't work properly... We cannot switch English to French and reverse. The old one version was better, to choose at the start.

And the last one, was you cannot change some things like creatures, items, races, units. You click on them and nothing appear or they are missing. Maybe a conflict with the French translation?

**Comment by Berzetoberze** [comment]
I have forgot something to add!

The Editor-Alpha-9-Snapshot don't have this bugs I have mentioned before.  But for the "Added races images" is still work in progress I think, let me help a little.

I have check all the categories, you can change or put the images of the races like this:
Dragons = Fyrmir
Developers = Skeleton
Citizen of Empyria = Turban woman
Demons and Titans = Ulather
Ice Elves = Ice elves (like the archer or guard, you choose)
Shaal Duhn Elves = Shar Arduin
Ghosts and others = Sirkah
Ghosts 2 = Belial
Winterguard = Aileen
Empyria Guards = Bronze Giant
Grim Reaper = no image
Hallits = Thane
Hallits 2 = Lord Jonir
Humans = Stellar Mage
Named Humans 2 = Caele
Named Humans = Farlorn
Player = Irfit
Kathai = Khal
The Fist = Orc
Briar = Briar's Elite
Red Legion = Halicor
Leonidars = Leonidar knight
Mages (maybe change the name, because it's not appropriate?) = Fial darg
Minotaur and orc = orc
Birds= no image
Order of the Dawn = Order knight
Orcs = inferno
Peasant = no image 
Sha = You choose like lancer or priest, for the moment it's a skeleton. 
Sharok = no image
Shial elves = Guard
Trolls = Stronghold
Uroks = Frang
Wolf for release = no image

**Comment by Hokan-Ashir** [comment]
Thanks for your response @Berzetoberze, right now I don't have much time to get Editor work correctly. But I appreciate your testing new version.
To be fair, you are the forth one, who noticed that 9th alpha gone mauvais
I'll check it out, I promise :)

**Comment by Berzetoberze** [comment]
Hey welcome back! =D

No problem, it's always a honour to help you to improve this editor! ;)

Check all my branch, I have make a big update for my return!

**Comment by Hokan-Ashir** [comment]
Hi @Berzetoberze, check out new version of alpha-9, I've fixed it a bit, but just a bit. Completely ignored reestablishing races images, cause that's just cosmetic change, but fixed search, opening items with quotes in names (like Asassin's Bow), opening and reopening .cff file problems with encoding (might have issues with french language)

So, I'm just wanna to ask you to check what can be changed =)

Sorry for slow response, just don't have mood and time to get my hands over editor

Yeah, I also merged almost all your localization

**Comment by Berzetoberze** [comment]
Hi my friends,

I'm very busy for the moment because I make a house moving. I go to the south for search some sun and warm ;)

Soo again, please standby =P

**Comment by Hokan-Ashir** [comment]
Le français garçon qui recherche par nouvelle chaude place? =)

Pas problem, mon ami, prends ton temps)

Excuse moi mon français, je just peux un peu practice ))

**Comment by Berzetoberze** [comment]
Let me correct this:

Le garçon français cherche une nouvelle place pour avoir chaud?
No mistake for the second sentence, congratulations! Just a small one, problem = problème in French.
Excuse moi pour mon français, je m'entraîne juste un peu.

I have already the keys for my new home, now I have to pick up my stuff and move it to the south. We have a small problem, I don't have internet there, I must go to my sister to catch the network! Maybe in the future I will pay an internet subscription but I'm not sure. Don't worry, I will notice you if I can.

And don't forget to notice me for new releases and misc. I'm always glad to help for your job.

**Comment by Hokan-Ashir** [comment]
Oh, I thought you're native french man =)

Sounds like you've moved in a very warm and not so urbanized country with melting wires =)

Nah, that's totally fine, don't worry about new releases, I'm switched to other deeds and editor in "support" state, nothing to actually add, maybe to fix something. But I don't get any responses from my fellas, and cause I'm kinda limited with testing (just in IDE) I'm keeping with "new issues? hm, I'll fix them and then go somewhere else" scenario

**Comment by Berzetoberze** [comment]
Yes I am! This is why I can make the French translation at the fittest!

Yes, unfortunately, when I was in the North I was in the countryside, It's very calm and relaxing.  But too difficult to join the city... 2 hours just to join the nearest town... It's difficult!
Now I'm in the heart of the city, Near of all markets, post office, misc...

Something new since november? I'm very busy for the moment. Sorry for the very late answer!

**Comment by Hokan-Ashir** [comment]
Yet still internet unites people, huh?)
Though, it must be tedious without IRL communications, cut from all other moving and flexing world

Nah, no news, just one little quick fix and new question about adding new items, which is impossible
I'm just tracking new bugs and supporting project)

**Comment by Berzetoberze** [comment]
Hey buddy! :)

Don't understand your two first sentences...?

Ok thanks!
I think in November or December I can come back for help you again but I'm not sure for the moment. Still busy for the moment. I try to stay tuned but it's hard ^^"
